src/server/graphql/models/makeChangefeedHandler.js

```diff
@@ -49,7 +50,7 @@ const handleRethinkUpdate = (doc, path) => {
     removeKeys
   };
   if (path) {
-    payload.path = `${path}[${payload.id}]`;
+    payload.path = `${path}[${docId}]`;
```

Comment:
when we update a single object in a changefeed, we'll send down something like `/teamMembers[123]` to tell it that we're updating the document where `id === 123`.